src/entitysdk/models/memodel.py

```diff
@@ -0,0 +1,96 @@
+"""Simulatable neuron model."""
+
+from typing import Annotated
+
+from app.db.types import ValidationStatus
```

Comment:
You can either use a str for the status, or define a StrEnum in `entitysdk.typedef`